tests/integration/api/test_feature_correctness.py

```diff
@@ -251,6 +319,90 @@ def feature_parameters_fixture(source_type):
     return parameters
 
 
+@pytest.mark.parametrize("source_type", ["snowflake", "spark"], indirect=True)
+def test_feature_with_target(event_table, observation_set, transaction_data_upper_case):
```

Comment:
Curious how long it takes to run this test? If the purpose of the test is to check that the end to end workflow involving both target and features work, not sure if we really need to test using this large observation set. It seems the correctness aspect is already covered by other tests.

Reply:
hm it's about 20th slowest - about 15-20s.

Reply:
I think it would be nice if the test can be made smaller, maybe 10 rows or so of observation set.